iib/web/api_v1.py

```diff
@@ -92,6 +92,8 @@ def add_bundles():
             request.id,
             payload.get('from_index'),
             payload.get('add_arches'),
+            payload.get('cnr_token'),
```

Comment:
Let me do some more digging and get back to you on this.

No, it isn't stored permanently.

Reply:
It could get logged though. Let's use this mechanism to hide it:
https://docs.celeryproject.org/en/stable/userguide/tasks.html#hiding-sensitive-information-in-arguments